### PR TITLE
CDN Link

### PR DESCRIPTION
**Pull request by roccomuso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,14 @@ $ npm install --save eventemitter3        # npm
 $ component install primus/eventemitter3  # Component
 $ bower install eventemitter3             # Bower
 ```
+## CDN
+
+Recommended CDNs:
```
Comment (lpinca) [inline README.md]: Should we mention that there isn't a UMD wrapper?

**Comment by lpinca** [comment]
I think it makes sense to follow the workflow suggested in https://unpkg.com/.
I'll make the npm package have a `umd` directory where a minified and UMD wrapped version can be found.

**Comment by roccomuso** [comment]
@lpinca cool :+1:

File: README.md (hunk 1 above)
Comment (roccomuso) [inline README.md]: yes definetly.

**Comment by lpinca** [comment]
@roccomuso what should be in your opinion the standalone name? I would go `EventEmitter3`.

**Comment by roccomuso** [comment]
@lpinca, I agree on `EventEmitter3`

**Comment by lpinca** [comment]
It seems to work: https://unpkg.com/eventemitter3@latest/umd/eventemitter3.min.js.
Can you please update the PR when you have time?

Thanks.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,14 @@ $ npm install --save eventemitter3        # npm
...
+
+```text
+https://cdnjs.cloudflare.com/ajax/libs/eventemitter3/2.0.2/index.min.js
```
Comment (lpinca) [inline README.md]: Should we remove this? There is no UMD bundle and unpkg uses cloudflare so maybe it's better to only keep unpkg.

Comment (roccomuso) [inline README.md]: yep

**Comment by lpinca** [comment]
Thanks!

**Pull request merged**